src/components/Player/Player.tsx

```diff
@@ -16,14 +20,28 @@ interface PlayerProps {
 }
 
 const Player: React.FC<PlayerProps> = ({ painting, resources }) => {
-  const viewerState: any = useViewerState();
-  const { configOptions } = viewerState;
-
   const playerRef = React.useRef(null);
   const isAudio = painting?.format?.includes("audio/");
```

Comment:
Not your code here but I'm wondering if this should be...
```suggestion
  const isAudio = painting?.type === "Sound";
```

Reply:
Yea, sounds good to me.  Guessing the original line is looking at some form of `mime type`, but an authored `painting.type` seems like it should be trusted more eh?